src/browser/components/UpdaterPage.jsx

```diff
@@ -0,0 +1,53 @@
+const React = require('react');
+const propTypes = require('prop-types');
+const {Button, Navbar} = require('react-bootstrap');
+
+function UpdaterPage(props) {
+  return (
+    <div>
+      <Navbar>
+        <h1 className='UpdaterPage-heading'>{'New update is available'}</h1>
```

Comment:
Do these strings need to be translated into the users' language?

Reply:
Not necessary because the desktop app doesn't have any localization system for now.